tests/app/questionnaire/test_rules.py

```diff
@@ -184,20 +184,20 @@ def test_evaluate_rule_checkbox_contains_all(self):
         self.assertFalse(evaluate_rule(when, []))
 
     def test_evaluate_rule_checkbox_contains_any(self):
-        when = {'values': ['a', 'b'], 'condition': 'contains any'}
+        when = {'values': ['aaa', 'bbb'], 'condition': 'contains any'}
```

Comment:
Maybe throw in a few extra characters here so it tests more cases? For example spaces, apostrophes and question mark?